### PR TITLE
Feature/about me

### PR DESCRIPTION
**Pull request by rangerisrael**

### CODE REVIEW

File: frontend/mocks/db.json [hunk 1]
```diff
@@ -1,7 +1,20 @@
 {
   "profile": {
+    "id": 1,
     "name": "Carlo Gino Catapang",
     "alias": "Pogi",
-    "age": 3
+    "age": 3,
+    "profile": "../assets/img/profile.png",
```
Comment (carlogino) [inline frontend/mocks/db.json:7]: change this to image

File: frontend/src/app/pages/about-me/about-me.component.scss [hunk 2]
```diff
@@ -0,0 +1,7 @@
+* {
+  margin: 0;
+}
```
Comment (carlogino) [inline frontend/src/app/pages/about-me/about-me.component.scss:3]: Remove this. This is already handled by our css normalizer.

File: frontend/src/app/pages/about-me/about-me.component.scss [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+
+.items {
+  padding: 2rem;
```
Comment (carlogino) [inline frontend/src/app/pages/about-me/about-me.component.scss:6]: Do not user hardcoded value. import from scss variables then use the appropriate variable. e.g. $padding-small

File: frontend/src/app/pages/about-me/profiles/profiles.component.ts [hunk 4]
```diff
@@ -0,0 +1,18 @@
+import { Component, OnInit } from '@angular/core';
+import { ApiService } from 'src/app/services/api.service';
+import { Profiles } from 'src/app/models/profile-model';
+
+@Component({
+  selector: 'app-profiles',
+  templateUrl: './profiles.component.html',
+  styleUrls: ['./profiles.component.scss'],
```
Comment (carlogino) [inline frontend/src/app/pages/about-me/profiles/profiles.component.ts:8]: Can you add changeDetection OnPush here?

File: frontend/src/app/pages/about-me/testimonial/testimonial.component.scss [hunk 5]
```diff
@@ -0,0 +1,21 @@
+.item {
+  margin-top: 12%;
+  display: flex;
+  flex-direction: column;
+}
+
+div p {
```
Comment (carlogino) [inline frontend/src/app/pages/about-me/testimonial/testimonial.component.scss:7]: Do not use html element in defining styles. Use css class.

File: frontend/src/app/pages/about-me/testimonial/testimonial.component.scss [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+  font-size: 1.15rem;
+  margin-bottom: -10%;
+  letter-spacing: 1.5px;
+  text-align: center;
+  align-items: center;
+  padding-left: 10%;
+  padding-right: 10%;
+}
+
+div > p:last-child {
```
Comment (carlogino) [inline frontend/src/app/pages/about-me/testimonial/testimonial.component.scss:19]: same here

File: frontend/src/app/pages/about-me/testimonial/testimonial.component.ts [hunk 7]
```diff
@@ -0,0 +1,18 @@
+import { Component, OnInit } from '@angular/core';
+
+import { ApiService } from 'src/app/services/api.service';
+import { Profiles } from 'src/app/models/profile-model';
+
+@Component({
+  selector: 'app-testimonial',
+  templateUrl: './testimonial.component.html',
+  styleUrls: ['./testimonial.component.scss'],
```
Comment (carlogino) [inline frontend/src/app/pages/about-me/testimonial/testimonial.component.ts:9]: add changeDetection OnPush

File: frontend/src/app/pages/about-me/testimonial/testimonial.component.ts [hunk 8]
```diff
@@ -0,0 +1,18 @@
...
+})
+export class TestimonialComponent implements OnInit {
+  testimonial: Profiles;
+  constructor(private apiService: ApiService) {}
```
Comment (carlogino) [inline frontend/src/app/pages/about-me/testimonial/testimonial.component.ts:13]: change to `private readonly apiService: ApiService`

File: frontend/src/app/pages/about-me/testimonial/testimonial.component.ts [hunk 9]
```diff
@@ -0,0 +1,18 @@
...
+
+  ngOnInit(): void {
+    this.apiService.getProfile().subscribe((profile: Profiles) => (this.testimonial = profile));
```
Comment (carlogino) [inline frontend/src/app/pages/about-me/testimonial/testimonial.component.ts:16]: If possible use observables here

File: frontend/src/app/pages/about-me/themes/themes.component.scss [hunk 10]
```diff
@@ -0,0 +1,12 @@
+* {
+  margin: 0;
+  padding: 0;
+}
```
Comment (carlogino) [inline frontend/src/app/pages/about-me/themes/themes.component.scss:4]: remove this

File: frontend/src/app/pages/about-me/themes/themes.component.scss [hunk 11]
```diff
@@ -0,0 +1,12 @@
...
+
+.themes {
+  display: flex;
+}
+
+div > .item {
```
Comment (carlogino) [inline frontend/src/app/pages/about-me/themes/themes.component.scss:10]: do not user html element, use css classes

File: frontend/src/app/services/api.service.ts [hunk 12]
```diff
@@ -13,4 +13,8 @@ export class ApiService {
   get fetchProfile() {
     return this.httpClient.get(environment.apiURL + '/profile') as Observable<Profile>;
   }
+
+  getProfile() {
```
Comment (carlogino) [inline frontend/src/app/services/api.service.ts:17]: reuse fetchProfile, you can change to normal method if you want

File: frontend/src/index.html [hunk 13]
```diff
@@ -11,6 +11,11 @@
       rel="stylesheet"
     />
     <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet" />
+    <link
+      href="https://fonts.googleapis.com/css?family=Roboto:300,400,500&display=swap"
+      rel="stylesheet"
+    />
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet" />
```
Comment (carlogino) [inline frontend/src/index.html:18]: Remove this duplicate code

File: frontend/src/styles.scss [hunk 14]
```diff
@@ -41,3 +41,12 @@ body {
...
 }
+
+html,
+body {
+  height: 100%;
+}
+body {
+  margin: 0;
+  font-family: Roboto, 'Helvetica Neue', sans-serif;
+}
```
Comment (carlogino) [inline frontend/src/styles.scss:52]: Remove all of this

File: frontend/angular.json [hunk 15]
```diff
@@ -169,6 +175,7 @@
   },
   "defaultProject": "frontend",
   "cli": {
-    "defaultCollection": "@ngrx/schematics"
+    "defaultCollection": "@ngrx/schematics",
+    "analytics": false
```
Comment (carlogino) [inline frontend/angular.json:179]: remove this

File: frontend/src/app/pages/about-me/about-me.component.html [hunk 16]
```diff
@@ -1 +1,5 @@
-<p>about-me works!</p>
+<div class="items">
```
Comment (carlogino) [inline frontend/src/app/pages/about-me/about-me.component.html:1]: use semantics tags for this

File: frontend/src/app/pages/about-me/profiles/profiles.component.html [hunk 17]
```diff
@@ -0,0 +1,17 @@
+<div *ngIf="person">
+  <div class="item">
```
Comment (carlogino) [inline frontend/src/app/pages/about-me/profiles/profiles.component.html:2]: use semantics tags for this. see https://www.tracylum.com/blog/2015-12-21-the-benefits-of-semantic-html/

**Review by codegino — CHANGES_REQUESTED** [review]
Changes needed

File: frontend/mocks/db.json (hunk 1 above)
Comment (rangerisrael) [reply frontend/mocks/db.json:7]: anu po ung ibig sabihin nyu sir d2? ung profile property path sir?  babaguhin ko po ung img to image

**Review by codegino — APPROVED** [review]

**Comment by carlogino** [comment]
Good job boss

**Pull request merged**